.github/workflows/release.yml

```diff
@@ -1,26 +1,38 @@
 on:
-  push:
-    branches:
-      - release/v*
+  release:
+    types:
+      - created
+
 jobs:
   aws-release:
-    name: Release zip file for lambda deployment
+    name: Release zip file for Lambda deployment
     runs-on: ubuntu-latest
-    needs: bump-version
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Set env
-        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
You could still use the `GITHUB_ENV` for the `ZIP_ASSET_NAME`, to avoid the duplicate definition you have below.